planner/core/testdata/integration_suite_out.json

```diff
@@ -5045,11 +5046,11 @@
         "SQL": "desc format = 'brief' select * from t right join (select id-2 as b from t) A on A.b=t.id",
         "Plan": [
           "HashJoin 12487.50 root  right outer join, equal:[eq(test.t.id, Column#9)]",
-          "├─TableReader(Build) 9990.00 root  data:Selection",
-          "│ └─Selection 9990.00 cop[tiflash]  not(isnull(test.t.id))",
+          "├─Projection(Build) 10000.00 root  minus(test.t.id, 2)->Column#9",
+          "│ └─TableReader 10000.00 root  data:TableFullScan",
           "│   └─TableFullScan 10000.00 cop[tiflash] table:t keep order:false, stats:pseudo",
-          "└─Projection(Probe) 10000.00 root  minus(test.t.id, 2)->Column#9",
```

Comment:
Expected, the current build-side has the smaller row-size which results in a smaller hash-build-cost.